help/change-your-language.md

```diff
@@ -19,6 +19,18 @@ messages you receive.
 
 1. Click **Reload**.
 
+{tab|mobile}
+
+{!mobile-profile-menu.md!}
+
+1. Tap **Settings**.
+
+1. Tap **Language**.
+
+1. Use the search tool or scroll through the list of languages.
+
+1. Select a language.
+
```

Comment:
How about adding an instruction for how to get out of the menu? Something like:

1. Tap the **arrow** (`<icon>`) in the upper left to exit the language selection menu.